lib/app_funcs.sh

```diff
@@ -60,6 +60,17 @@ function compile_app() {
   cd - > /dev/null
 }
 
+function post_compile_hook() {
+  cd $build_path
+
+  if [ -f post_compile.sh ]; then
+    output_section "Executing post_compile.sh"
+    ./post_compile.sh || exit 1
```

Comment:
@ericmj what you think about adding the `post_compile` to `elixir_buildpack.config`?
Like:
```sh
# Execute bash script after App installation
post_compile=deploy/post_app.sh
```
and then change this to:
```elixir
if [ -f $post_compile ]; then
```